supertokens_python/recipe/emailpassword/utils.py

```diff
@@ -185,11 +172,13 @@ def __init__(self,
                  form_fields_for_password_reset_form: List[NormalisedFormField],
                  form_fields_for_generate_token_form: List[NormalisedFormField],
                  get_reset_password_url: Callable[[User, Dict[str, Any]], Awaitable[str]],
-                 create_and_send_custom_email: Callable[[User, str, Dict[str, Any]], Awaitable[None]]):
+                 create_and_send_custom_email: Union[Callable[[User, str, Dict[str, Any]], Awaitable[None]], None] = None):
         self.form_fields_for_password_reset_form = form_fields_for_password_reset_form
         self.form_fields_for_generate_token_form = form_fields_for_generate_token_form
         self.get_reset_password_url = get_reset_password_url
-        self.create_and_send_custom_email = create_and_send_custom_email
+        self.create_and_send_custom_email = None
```

Comment:
you should still assign the argument instead of None. the option should work until we remove the parameter.

Reply:
done